frontend/src/component/context/EditContext/EditContext.tsx

```diff
@@ -77,6 +84,7 @@ export const EditContext = () => {
 
     return (
         <FormTemplate
+            modal={modal}
```

Comment:
This allows the form to grow to the full height in the modal context, preventing this:

<img width="1904" height="1771" alt="Image" src="https://github.com/user-attachments/assets/ef9b3548-a1c0-438e-a161-54d42c64d4e7" />